packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -197,7 +197,7 @@ export function AutoCrud<TModel extends AbstractModel>({
     />
   );
 
-  return (
+  const editableGrid = (
```

Comment:
I'd suggest using `if` block here to avoid unnecessary creation of a JSX object in case form is not enabled. Something like:

```tsx
if (noNewButton !== true || (item && item !== emptyItem)) {
  // return created editable grid 
}

// return created mainSection
```